messaging/__init__.py

```diff
@@ -12,11 +16,12 @@
 
 assert MultiplePublishersError
 assert MessagingError
-assert toggle_fake_events
-assert set_fake_prefix
-assert get_fake_prefix
-assert delete_fake_prefix
-assert wait_for_one_event
+if platform.system() != "Darwin":
```

Comment:
I think a stub implementation would be nicer here. This PR is +30 lines, which is fine, but it would be nicer if those 30 lines were in a single file with single `if mac`.  currently, there's a little bit of added code all of the place with a bunch of `if mac`